modules/auth/user_form.go

```diff
@@ -27,6 +27,7 @@ type InstallForm struct {
 	SSHPort      int
 	HTTPPort     string `binding:"Required"`
 	AppUrl       string `binding:"Required"`
+	LogRootPath  string
```

Comment:
We want to have this optional right? So that if it's empty we default to system setting.